tests/dsl_unittest.py

```diff
@@ -1859,7 +1859,7 @@ def top(context):
                 async def example(self):
                     time.sleep(0.1)
 
-            with self.assertRaisesRegex(RuntimeError, "^Executing .+ took .+ seconds$"):
+            with self.assertRaisesRegex(SlowCallback, "^Executing .+ took .+ seconds$"):
```

Comment:
Tip: if you use multiple line messages here, eg:

```
Event loop got blocked for more than Xms.
Executing .... took ... seconds
```

The test runner properly cut the first line, and only show the long message at the end of the report.

Reply:
PS: It just occurred to me: the exception message could mention `asyncio.get_running_loop().slow_callback_duration = seconds` to inform users how to customize it if needed, just like the docs do.

Reply:
Not sure I understand what you mean here, are you suggesting adding "Event loop got blocked for more than ..." to the `SlowCallback` (`EventLoopLockUp` per your rename suggestion) error message?

Reply:
What I (miserably) tried to mean (sorry!) was that the error message should be as informative and educative as possible. I found myself in the past few days explaining several times why event loop lock ups are something important to surface, let's make this a job of the error message. I also got asked "what if I'm OK with 0.2s (not 0.1s)?" a few times, so, informing about setting `slow_callback_duration` (eg: using a before hook) would also be helpful.